prime-router/src/main/kotlin/transport/NullTransport.kt

```diff
@@ -26,7 +26,7 @@ class NullTransport : ITransport {
     ): RetryItems? {
         if (header.content == null) error("No content for report ${header.reportFile.reportId}")
         val receiver = header.receiver ?: error("No receiver defined for report ${header.reportFile.reportId}")
-        val msg = "Sending to Null Transport"
+        val msg = "Sending to Null Transport. File can be downloaded by Receiver until it expires."
```

Comment:
Is this a message that is ever viewed by our consumers, or just our team? If viewed by consumers "Sending to Null Transport" could be confusing (especially if it's not a developer).

Reply:
I'm pretty sure that's just in the DB for our logs. I can confirm that it doesn't show up on the History API, and I'm not sure what else might show that, since it's logged on the Action table.

Reply:
Right on. I'm very unfamiliar with what messages get viewed by just our engagement team or which might be seen by STLTs or other consumers so I just wanted to check.